RecoBTag/TensorFlow/src/tensor_fillers.cc

```diff
@@ -77,6 +77,84 @@ namespace btagbtvdeep {
     *(++ptr) = tag_info_features.z_ratio;
   }
 
+  void db_tensor_filler(tensorflow::Tensor & tensor,
+                         std::size_t jet_n,
+                         const btagbtvdeep::DeepDoubleCvLFeatures & features) {
```

Comment:
the implementation is identical to the filling of ```DeepDoubleBFeatures```
looks like this is not needed

Reply:
removed